tests/behat/behat-features/contentpool-client.feature

```diff
@@ -12,7 +12,7 @@ Feature: Contentpool client-side replication works.
     And I fill in "password" with "test"
 
   Scenario: Replication via drush works
-    Given I run drush cpc
+    Given I run drush cppull
     And I run drush cron
```

Comment:
the drush cron command should be removed here.
Let's allso add test coverage for "cpc + queue-run" to also work as cppull.